msys2 clang64 failing

cc @oxillo 

Looks like it is an issue with svgtiny / libparserutils. 
mingw32 and mingw64 are fine but clang64 is erroring with: `call to undeclared function 'N_ELEMENTS';  

```
libparserutils/src/charset/codecs/codec_ext8.c:133:18: error: call to undeclared function 'N_ELEMENTS'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
        for (i = 0; i < N_ELEMENTS(known_charsets); i++) {
```

Here is the job where it is failing 
https://github.com/openframeworks/apothecary/actions/runs/3138693992/jobs/5098309938

--- a/apothecary/formulas/svgtiny/svgtiny.sh
+++ b/apothecary/formulas/svgtiny/svgtiny.sh
@@ -45,7 +45,7 @@ function prepare() {
 		cp -r $FORMULA_DIR/vs2017 ./
 	else
 		# Apply patch for 'N_ELEMENTS() implicit declaration' for OSX and Emscripten
-		if [ "$TYPE" == "osx" ] || [ "$TYPE" == "ios" ]|| [ "$TYPE" == "tvos" ] || [ "$TYPE" == "emscripten" ] ; then
+		if [ "$TYPE" == "osx" ] || [ "$TYPE" == "ios" ]|| [ "$TYPE" == "tvos" ] || [ "$TYPE" == "emscripten" ] || [ "$TYPE" == "msys2" ] ; then
 			cd libparserutils
 			patch -up1 < $FORMULA_DIR/libparseutils.patch
 			cd ..

